internal/wasm/host.go

```diff
@@ -9,6 +9,16 @@ import (
 	"github.com/tetratelabs/wazero/internal/wasmdebug"
 )
 
+// Func is a function with an inlined type, typically used for NewHostModule.
```

Comment:
what does "inlined type" mean here?

Reply:
normally a module defines the function data in 3 places, the type section, the function section (points to type) then code section. This does the same thing in struct as allowed in the text format, where you define the type inline without needing to know its ultimate position in the type index namespace. this allows flexibility as we can declare it without any regards to what module it eventually is compiled into.

Reply:
got it!

Reply:
for others.. this part https://www.w3.org/TR/wasm-core-1/#abbreviations%E2%91%A6